kubeportal/settings.py

```diff
@@ -5,8 +5,8 @@
 
 
 class Common(Configuration):
-    VERSION = '0.5.4'
-    API_VERSION = 'v1.4.0'
+    VERSION = '0.6.0'
+    API_VERSION = 'v2.0.0'
```

Comment:
why 2.0.0? which old routes are affected?

Reply:
All. The User ID as part of the URL is gone.